.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug
+about: Create a bug report to help us improve
+title: ''
+labels: kind/bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+ **Expected behavior**
+A clear and concise description of what you expected to happen.
+
+ **To Reproduce**
+Steps to reproduce the behavior.
+
+ **Knative release version**
+
```

Comment:
It seems you need to add description along this line anyhow: 

  | 0s
-- | --
.github/ISSUE_TEMPLATE/bug.md:19: MD036 Emphasis used instead of a header